app/components/publish/providers/school_placements/explainer_component.html.erb

```diff
@@ -0,0 +1,16 @@
+<p class="govuk-body">
+  Add the schools you can offer placements in. A placement school is where candidates will go to get classroom
+  experience.
```

Comment:
The preposition at the end of the first sentence is pretty grating. Should it be: 
`Add the schools where you can offer placements`

Reply:
One for @RebeccaCryan when she returns

Reply:
Resolved in https://github.com/DFE-Digital/publish-teacher-training/pull/4525/commits/ce52388bf119954b884fd36f0f0cc63febff83ba